samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Uno.Toolkit.Samples.Shared.projitems

```diff
@@ -74,6 +74,9 @@
     <Compile Include="$(MSBuildThisFileDirectory)Content\Controls\VisualStateManagerExtensionsSamplePage.xaml.cs">
       <DependentUpon>VisualStateManagerExtensionsSamplePage.xaml</DependentUpon>
     </Compile>
+    <Compile Include="$(MSBuildThisFileDirectory)Content\Controls\ZoomContentControlSamplePage.xaml.cs">
```

Comment:
If we are adding a new Sample, would be better to remove or update the PhotoViewer sample which was used for the same purpose. What do you think @agneszitte ?

Reply:
what do you think @kazo0 ?

Reply:
For now we can just have that sample in the Toolkit Samples App and we can remove it from Uno.Samples at a later date